Voycar.Api.Web/Generic/Endpoint/Put/Single.cs

```diff
@@ -19,6 +19,20 @@ public override void Configure()
     {
         this.Put(typeof(TEntity).Name.ToLowerInvariant());
         this.Roles(this.roles);
+        Description(b => b
```

Comment:
Hier haben wir uns für entschieden immer ein "this." vorzuschreiben. Eigentlich sollte dich deine IDE auch gewarnt haben.